### ISSUE TITLE
[feat] Abilitiy to manage/delete tags

### ISSUE THREAD

**Issue by turnschuhadmin1337** [raimund-schluessler/inventory#563]
Hi,
is it possible to manage tags? E.g. for the usecase to remove typos from tags or after testing etc.?

Inside the "Tags" I can see all my tags so far, but some of them are old or testing and empty.

Thanks so much for your work.

**Comment by raimund-schluessler** [comment]
Indeed, that is a feature that is currently missing. At the moment, you can only add or remove tags from items, but they are still in the database an listed in the available tags. Editing or removing is not implemented. Related to #535.
